teaser/logic/buildingobjects/calculation/four_element.py

```diff
@@ -288,6 +288,8 @@ class FourElement(object):
         the ambient.
     ua_value_win : float [W/K]
         U-Value times outer wall area.
+    u_value_win : float [W/(m2K)]
+        U-Value of windows.
```

Comment:
I fixed that by mentioning it in description. Do you think that is sufficient or did you have in mind to rename that parameter?